CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Version History
     * All Version bumps are required to update this file as well!!
 ----
 
+* 18.0.0 Adding job search version 3 as additional call
```

Comment:
I'm on the fence. On the one hand it's backwards compatible as you pointed out. On the other hand it's (semi) brand new code paths. Anyone else wanting it to be a 17.X version instead?